deezer/client.py

```diff
@@ -185,6 +185,27 @@ def get_object(
                     object_t, object_id
                 )
             )
+
+        # fix deezer's truncation of items (e.g. tracks on large playlists)
+        if (
+            (
+                relation == "tracks"
+                or relation == "fans"
+                or relation == "albums"
+                or relation == "artists"
+                or relation == "playlists"
```

Comment:
Why add these restrictions on specific relations? Could we could do it whenever there is a `next` in the response, regardless of the relation being requested?

Reply:
I remember testing it without that and something going horribly wrong, but that was before `and limit not in kwargs` got inserted. I'll do some messing around and see if I can repro the same issue again, otherwise I'll take them out

Reply:
Still can't remember exactly why I left that in, but I think it had something to do with endpoints such as (and possibly only) `artist/<artist>/top`. I suppose I thought it was a different question whether artists' top songs should be listed beyond the API's default limit (without specifying a higher limit).

The benefits of following "next" URLs for requesting (e.g.) playlist tracks is fairly clear-cut in my opinion, as - if you ask for the tracks of a playlist - you expect all the tracks from that playlist to be returned. It's a bit more unclear as to whether all of an artist's top 100 songs (the API maximum) should be returned by default when making that call.